src/Qyl.OpenTelemetry.SemanticConventions.Incubating/README.md

```
@@ -0,0 +1,26 @@
+[![NuGet Qyl.OpenTelemetry.SemanticConventions](https://img.shields.io/nuget/v/Qyl.OpenTelemetry.SemanticConventions?label=Qyl.OpenTelemetry.SemanticConventions&color=0891B2)](https://www.nuget.org/packages/Qyl.OpenTelemetry.SemanticConventions/)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)